test/library/packages/Sort/RadixSort/simpleMerge.chpl

```diff
@@ -0,0 +1,53 @@
+proc merge(array:[] int, l: int, m: int, r:int){ 
+    
+    var i: int;
+    var j: int; 
+    var k: int; 
+    var n1 = m - l + 1; 
+    var n2 =  r - m; 
+  
+    var L:[1..n1] int;  
+    var R:[1..n2] int;
+
+    for i in 1..n1 { 
+        L[i] = array[l + i - 1]; 
```

Comment:
We should probably use a different variable name other than `i` to prevent shadowing the `i` from the outer scope.